api/server.py

```
@@ -0,0 +1,278 @@
+        txn = df_transaction[df_transaction['Transaction_Id'] == (transaction_id)]
+        
+
+        df_combined = pd.merge(txn_pred, txn, on='Transaction_Id', how='inner')
+        
+        if df_combined.empty:
+            return {"message": "No records found."}
+
+        return df_combined.to_dict(orient="records")[0]
+    
+    except FileNotFoundError:
+        return HTTPException(status_code=404, detail= f"error CSV file not found")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)